static-files/eclipse-che-logo.svg

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This one doesn't appear to be in use either?

Reply:
it was used to generate b64, I think we should keep those logos here alongside b64 master file so it's easier to track changes and reuse.
Although - I'm ok to delete them if you don't want that bundled here

Reply:
it's ok to keep them if they are expected, I was just making sure it wasn't an accidentally commited file.